pkg/karmadactl/cmdinit/utils/rbac.go

```diff
@@ -0,0 +1,89 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+// ClusterRoleFromSpec ClusterRole spec
+func ClusterRoleFromSpec(name string, rules []rbacv1.PolicyRule) *rbacv1.ClusterRole {
+	return &rbacv1.ClusterRole{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "rbac.authorization.k8s.io/v1",
+			Kind:       "ClusterRole",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   name,
+			Labels: map[string]string{"karmada.io/bootstrapping": "rbac-defaults"},
```

Comment:
Just a question: why do we need this label?

Reply:
For ClusterRole `cluster-proxy-admin` we don't need this labels, if necessary, we can put the labels to input parameters.

Reply:
ok

Reply:
fixed. @XiShanYongYe-Chang